### ISSUE TITLE
how to plot tensorboard with val and train data (not just train plot)

### ISSUE THREAD

**Issue by Adithia99** [facebookresearch/detectron2#1857]
## ❓ How to do something using detectron2
so im using tensorboard to ploting loss train ,mask, etc but it just train data. how to plot train and val in same plot in tensorboard?

Describe what you want to do, including:
1. what inputs you will provide, if any:
doing instance segmentation

2. what outputs you are expecting:
ploting loss val and train in same tensorboard ,so i can see the ploting is that overfitting or underfitting or fitting

## ❓ What does an API do and how to use it?
Please link to which API or documentation you're asking about from
https://detectron2.readthedocs.io/
load tensorboard

**Comment by ppwwyyxx** [comment]
There is no such thing called "validation loss" defined in detectron2. If you compute it somehow, https://detectron2.readthedocs.io/tutorials/training.html#logging-of-metrics can be used to save data to tensorboard.

If the trainer has evaluators properly configured, using `EVAL_PERIOD` can make it evaluate during training and save evaluation results to tensorboard.
